webapp/graphite/remote_storage.py

```diff
@@ -139,7 +139,7 @@ def fetch(self, startTime, endTime, now=None, result_queue=None):
       connection.request('POST', '/render/', query_string)
     else:
       connection.request('GET', '/render/?' + query_string)
-    response = connection.getresponse()
+    response = connection.getresponse(buffering=True)
```

Comment:
Here is quite easy fix for 2.6 support:
Maybe do something like
```
  try: # Python 2.7+, use buffering of HTTP responses
    response = connection.getresponse(buffering=True)
  except TypeError: # Python 2.6 and older
    response = connection.getresponse()
```
Assert will catch everything else.
Not sure how to patch line 81 though.